src/app/components/categories-details/categories-details.component.ts

```diff
@@ -171,6 +210,24 @@ export class CategoriesDetailsComponent {
       .subscribe();
   }
 
+  onCheckboxChange(storeId: string) {
+    let finalStoreIds: string[];
+    this.filters$
+      .pipe(
+        take(1),
```

Comment:
Do you need take(1) twice ?

Reply:
I'll check and delete if some is unnecessary in next PR